website/blog/2020-11-20-lithium.md

```diff
@@ -0,0 +1,204 @@
+---
+author: Tomasz Godzik
+title: Metals v0.9.6 - Lithium
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We're happy to announce the release of Metals v0.9.6, which mostly concentrates
+on adding support for the newly released Scala 2.13.4, but also adds some
+improvements for alreadye xisting features.
```

Comment:
```suggestion
improvements for already existing features.
```

Reply:
Tnaks! Fixed! Should have turned on the spellchecker before :sweat_smile: